Azure/sonic-buildimage
# [libteam] Add fallback support for single-member-port LAG

**- What I did**
* Allow the port to be selected if the LAG is configured
with fallback and port is in defaulted state due to missing
LACP PDUs from remote end
* Only enable port if LAG is admin up and the member port
is link up
**PLEASE note this pull request will only support LACP Fallback for single-member-port LAG as now. Since with this pull request, all member ports within LAG with fallback configured will be selected and active, which may not work for multiple member port LAG in some cases.**

**- How I did it**
* Add is_lacp_fallback_eligible check to lacp_port_selectable_state to make the port selectable if fallback is configured and port is in defaulted state

**- How to verify it**
* Connect the LAG to a server without LACP running. Check the LAG is still up on the switch side with fallback enabled.

**- Description for the changelog**




**- A picture of a cute animal (not mandatory but encouraged)**
